recipe/build.sh

```diff
@@ -1,9 +1,12 @@
-if [[ "`uname`" == "Linux" ]]
-then
-    export LDFLAGS="${LDFLAGS} -Wl,-rpath-link,${PREFIX}/lib"
+if [[ "$target_platform" == "amd64" || "$target_platform" == "win-64" ]]; then
```

Comment:
This `amd64` hack is because of https://github.com/conda-forge/vc-feedstock/issues/16

Reply:
See conda-forge/vc-feedstock#17